### DISCUSSION TITLE
Failed to process chunking: request entity too large

### DISCUSSION THREAD

**Discussion by Jeevanhm** [FlowiseAI/Flowise#3720] [General]
Has anyone faced this error "Failed to process chunking: request entity too large" while uploading multiple document to Docx File in Document store?

**Comment by AnantJain-Wangoes** [comment]
Same problem, any solution?

**Comment by e8-LiuPeng** [comment]
When I set the docker compose env FLOWISE_FILE_SIZE_LIMIT=500mb. It works for me.
